x-pack/plugins/logstash/server/models/cluster/cluster.test.ts

```diff
@@ -16,7 +15,7 @@ describe('cluster', () => {
 
       it('returns correct Cluster instance', () => {
         const cluster = Cluster.fromUpstreamJSON(upstreamJSON);
-        expect(cluster.uuid).to.be(upstreamJSON.cluster_uuid);
+        expect(cluster.downstreamJSON).toEqual({ uuid: upstreamJSON.cluster_uuid });
```

Comment:
I don't know TS. I assume this change is needed because it has something to do with how TS works?

Reply:
no, I can revert them or make `uuid` as a part of public interface

Reply:
I see. On one hand I assume it's better not to make `uuid` part of the public interface to keep the public interface smaller. OTOH I think the fewer changes we have in tests during the migration, the more confidence we can have that the migration didn't break anything.

I'll defer to your judgement on this.